ql/src/java/org/apache/hadoop/hive/ql/parse/repl/CopyUtils.java

```diff
@@ -401,6 +404,18 @@ private boolean isSourceFileMismatch(FileSystem sourceFs, ReplChangeManager.File
     return false;
   }
 
+  @VisibleForTesting
+  private void runTestOnlyExecutions() throws IOException {
```

Comment:
Wondering if this logic can be. moved to test itself

Reply:
Yahh, My first try was to do so, I thought of using PowerMock, But MiniDfs has issue with it. Which is part of Hadoop, We can't bother that.
The most I could pull out is a Callable into the test. So to avoid the delete or FS operations here, and in case in future that can be used later as well. Let me know if there is any other way out. :-)